manager/state/raft/raft_test.go

```diff
@@ -286,6 +290,7 @@ func TestRaftLeaderLeave(t *testing.T) {
 	t.Parallel()
 
 	nodes, clockSource := raftutils.NewRaftCluster(t, tc)
+	defer raftutils.TeardownCluster(t, nodes)
```

Comment:
Should I remove that line? Because I think that it should be in defer and for all functions.